### PR TITLE
Remove last traces of has-emoji class

### PR DESCRIPTION
**Pull request by mrsdizzie**
Now that emojify.js has been removed, get rid of all instances of has-emoji class that was only used for that. Support for rendering shortcodes remains in all of these places so it should still work the same.

### CODE REVIEW

**Comment by lafriks** [comment]
Does functions `RenderCommitMessageLinkSubject` and `RenderCommitMessage` does emoji rendering also?

**Comment by mrsdizzie** [comment]
> Does functions `RenderCommitMessageLinkSubject` and `RenderCommitMessage` does emoji rendering also?

Yep:

https://github.com/go-gitea/gitea/blob/4974b7c120102b49548197e58c7a58181ba52170/modules/markup/html.go#L200-L210

(same for subject one below that too)

**Review by lafriks — APPROVED** [review]

File: templates/explore/repo_list.tmpl [hunk 1]
```diff
@@ -29,7 +29,7 @@
 				</div>
 			</div>
 			<div class="description">
-				{{if .DescriptionHTML}}<p class="has-emoji">{{.DescriptionHTML}}</p>{{end}}
+				{{if .DescriptionHTML}}<p class=>{{.DescriptionHTML}}</p>{{end}}
```
Comment (zeripath) [inline templates/explore/repo_list.tmpl]:
```suggestion
				{{if .DescriptionHTML}}<p class>{{.DescriptionHTML}}</p>{{end}}
```

Comment (6543) [inline templates/explore/repo_list.tmpl]:
```suggestion
				{{if .DescriptionHTML}}<p>{{.DescriptionHTML}}</p>{{end}}
```
why not remove it completly ?

Comment (mrsdizzie) [reply templates/explore/repo_list.tmpl]: thanks fixed -- wish there was a linter for templates!

**Review by 6543 — APPROVED** [review]
I did not grep recursive for 'has-emoji' jet. But this should catch some ;)

**Comment by silverwind** [comment]
I take it the cases where only the class was removed and nothing added were already rendering correctly?

**Comment by mrsdizzie** [comment]
> I take it the cases where only the class was removed and nothing added were already rendering correctly?

yep https://github.com/go-gitea/gitea/pull/11263#issuecomment-622125682 as an example

**Review by 6543 — CHANGES_REQUESTED** [review]
```
$ rg 'has-emoji' options/locale/locale_en-US.ini
868:issues.add_label_at = added the <div class="ui label has-emoji" style="color: %s\; background-color: %s">%s</div> label %s
869:issues.remove_label_at = removed the <div class="ui label has-emoji" style="color: %s\; background-color: %s">%s</div> label %s
```

**Comment by 6543** [comment]
had time to rg it ;) :point_up:

**Comment by mrsdizzie** [comment]
> $ rg 'has-emoji' options/locale/locale_en-US.ini
> 868:issues.add_label_at = added the <div class="ui label has-emoji" style="color: %s\; background-color: %s">%s</div> label %s
> 869:issues.remove_label_at = removed the <div class="ui label has-emoji" style="color: %s\; background-color: %s">%s</div> label %s

ah sneaky : ) rending for this was already fixed in template code but have removed class name from translations file

**Review by 6543 — APPROVED** [review]
:+1:

**Pull request merged**